File: 프로그래머스/0/120849. 모음 제거/모음 제거.cs
using System;

public class Solution 
{
    public string solution(string my_string) 
    {
        string[] vowel = {"a", "e", "i", "o", "u"};
        
        string result = "";
        for(int i=0; i<my_string.Length; i++)
        {
            bool isVowel = false;
            for(int j=0; j<vowel.Length; j++)
            {
                if(my_string[i].ToString() == vowel[j])
                {
                    isVowel = true;
                }
            }
            if(isVowel)
            {
                continue;
            }
            result += my_string[i].ToString();
        }
        return result;
    }
}

File: 프로그래머스/0/120849. 모음 제거/모음 제거.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

string solution(string my_string) 
{
    vector<char> vec = {'a','e','i','o','u'};
    string result = "";
    for(char &c : my_string)
    {
        if(find(vec.begin(), vec.end(), c) == vec.end())
        {
            result += c;
        }
    }
    return result;
}